docs/running_vio_locally.md

```diff
@@ -0,0 +1,70 @@
+# Running VIO locally
+
+In order to use VIO locally we are going to start 3 modules in 3 different terminals. Some of them will need conda installed.
+The most direct way to install conda on MacOS is via Homebrew:
+```
+brew update
+brew install --cask miniconda
+```
+
+## Running the edge model serving
+The edge model serving is the module that is going to do the inference computing using the stored models. It is called
+by the edge orchestrator.
+
+You can follow the conda environment installation from the 
+[edge model serving's ReadMe](../edge_model_serving/tflite_serving/README.md) file. Once it is done you can start the 
+server using the make command.
+
+```
+make run_tflite_serving
+```
+
+## Running the edge orchestrator
+The edge orchestrator will administrate the configuration, images captures, storage and communication with the edge
+models for inference then applying business rules.
+The following commands will create a package of the orchestrator environment as [described here](edge_orchestrator.md)
+```
+cd edge_orchestrator
+make conda_env
+make install
+pip install -e .[dev]
+```
+
+It may be required for you to change the default profile of VIO depending on your needs. 
```

Comment:
Instead of changing the default config, what about creating a new one adapted to your need ?

Reply:
on pourrait simplifier les configs existantes (virer le mongo, le in-memory storage)

Reply:
Pourquoi pas

Reply:
je vais créer un ticket pour logger les idées